examples/batch_produce.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import random
+
+from aiokafka.producer import AIOKafkaProducer
+
+loop = asyncio.get_event_loop()
+
+
+async def send_many(num):
+    topic  = "my_topic"
+    producer = AIOKafkaProducer(loop=loop)
+    await producer.start()
+
+    batch = producer.create_batch()
+
+    i = 0
+    while i < num:
+        payload = "Test message %d" % i
+        if not batch.append(key=None, value=payload.encode(), timestamp=None):
```

Comment:
Set explicit encoding for `encode()`. That way it will be self-documentary to what it does for people with less python knowledge.